src/modals/share-achievements/index.tsx

```
@@ -0,0 +1,145 @@
+      width={495}
+    >
+      <div className="achievements-info">
+        <div className="logo">
+          <Vultisig className="shape" />
+          <span className="name">Vultisig</span>
+        </div>
+        <div className="share-vult">
+          <div className="share-info">
+            <span
+              className="title"
+              dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)